Update Nest documentation for Google Home notification setings

gtalarico opened the pull request:
## Proposed change.

Proposal for adding more details on how google home app settings can affect HA ability to use receive camera events.
See community discussion:
https://community.home-assistant.io/t/google-nest-camera-not-sending-events/631491/5

PR is in progress and created to request feedback from maintainers. 
If we like this, I am hoping we can get folks that own some of these other cameras to fill confirm and fill in settings for other models.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository:  n/a
- This PR fixes or closes issue: fixes # n/a

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

gtalarico committed:
Update nest.markdown for home app settings

gtalarico committed:
Update nest.markdown

gtalarico committed:
Add screenshot

allenporter commented:
Thank you for taking the time to make these documentation improvements! Very much appreciated! 

One thing we always try to balance is how much to be helpful vs how much maintenance to do in case things change. I also came across https://support.google.com/googlenest/answer/9230439 and I wonder how much of that support article is applicable here to reference? Maybe we can keep some of the structure here, but reduce some of the detail.

gtalarico commented:
That's a fair point @allenporter 
I wasn't sure if there was a lot of nuance variations between them different camera models 
but agree to start w/ something simple

gtalarico committed:
Remove device table

allenporter commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,38 @@ This feature is enabled by the following permissions:
+The Google Home App Notifications settings control not only which notifications are sent to your phone,
+but they also affect what gets published to the Pub/Sub feed.
+
+For example, if you enable “away-only notifications”, HASS will only receive events when your phone is away from home.
```
```suggestion
For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
```

gtalarico commented:
I added a link the troubleshooting guide and removed the "per device" matrix.

I also added a some notes about the "decoupling" of device feed and Pub/Sub feed, and included a 
a table explaining how these key app settings that affect outcome. Do you think this is too much?

I find it very counter intuitive that these settings can cause events to not published to p/s feed, 
especially when google home app settings show the events in its internal feed.

PS: Here is another similar report - I am hoping adding these here could reduce issues and troubleshooting time for users.
https://github.com/home-assistant/core/issues/58482#issuecomment-1175586650

allenporter commented:
I think its a good middle ground how you have it now. Yes, I agree it is confusing and I think this will definitely help a lot of users.

gtalarico commented:
Sound good. Let me address the CI checks and will tag you for review

gtalarico committed:
PR comments

gtalarico commented:
@allenporter all 🟢 , please let me know if you have any additional feedback or change requests

allenporter commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
+| ------------------------ | :-----------------------------------------------------------------------------------: |
+| Notifications: Push      | Required for detection event to be published                                          |
+| Notifications: Away-Only | Events will only be published when owner is detected as away from home                |
+| Seen: Motion             | Only `motion` events will be published                                                |
```
Is it "Only" that those events will be published? Seems like this option isn't exclusive with the others

gtalarico replied:
Oops that's not correct.
Revsion here https://github.com/home-assistant/home-assistant.io/pull/29576/commits/f7060e1dab7ebadcd07c4cc066c74a4e0356b8a4

gtalarico committed:
revise seen: notes

allenporter reviewed (dismissed):
Looks good, thank you.

gtalarico commented:
Great🔥  
Are PRs merged in batches by maintainers? 
Do we need to update tags ? I think `in-progress` and `needs-rebase` can be removed

c0ffeeca7 commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
+
+For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
+
+Another thing that may not be intuitive, is that seeing the event your device history does not mean it's was published to the feed.
```
```suggestion
Another thing that may not be intuitive, is that seeing the event in your device history does not mean it was published to the feed.
```

c0ffeeca7 commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
+For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
+
+Another thing that may not be intuitive, is that seeing the event your device history does not mean it's was published to the feed.
+However, if you are getting push notifications the settings are likely working.
```
```suggestion
However, if you are getting push notifications, the settings are likely working.
```

c0ffeeca7 commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
+Another thing that may not be intuitive, is that seeing the event your device history does not mean it's was published to the feed.
+However, if you are getting push notifications the settings are likely working.
+
+Note: The exact settings and affect to feed  may vary by camera model or app version.
```
```suggestion
Note: The exact settings and effect they have on the feed may vary by camera model or app version.
```

c0ffeeca7 commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
+Note: The exact settings and affect to feed  may vary by camera model or app version.
+
+
+If you are still not getting notifications you can read this [troubleshooting guide from Google]<!-- textlint-disable -->
```
```suggestion
If you are still not getting notifications, you can read this [troubleshooting guide from Google]<!-- textlint-disable -->
```

c0ffeeca7 commented on source/_integrations/nest.markdown:
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
+| Google Home App Setting  | Notes                                                                                 |
+| ------------------------ | :-----------------------------------------------------------------------------------: |
+| Notifications: Push      | Required for any detection event to be published                                      |
+| Notifications: Away-Only | Events will only be published when user is detected as away from home                 |
```
```suggestion
| Notifications: Away-Only | Events will only be published when a user is detected as away from home                 |
```

c0ffeeca7 commented on source/_integrations/nest.markdown:
```diff
@@ -575,6 +606,8 @@ logger:
 
 - It is recommended to let Home Assistant create the Pub/Sub subscription for you. However, if you would like more control you can enter a `subscriber_id` in the configuration. See [Subscribe to Events](https://developers.google.com/nest/device-access/subscribe-to-events) for more instructions on how to manually create a subscription and use the full subscription name in the Home Assistant config ...
 
+- *Not receiving camera motion and person events*: assuming the integration is correctly configured (e.g. oauth and SDM API is setup correctly, you can see camera streams, and permissions are correctly set in [Partner Connections Manager](https://nestservices.google.com/partnerconnections)), but you are still not seeing events, it's possible you need to adjust Google Home App settings. See the [C ...
```
```suggestion
- *Not receiving camera motion and person events*: assuming the integration is correctly configured (for example, the oauth and SDM API are set up correctly, you can see camera streams, and permissions are correctly set in [Partner Connections Manager](https://nestservices.google.com/partnerconnections)): If you are then still not seeing events, it's possible you need to adjust the Google Home App settings. Refer to the [Cameras: Automation: Google Home App Settings](#google-home-app-settings) for details.
```

c0ffeeca7 committed:
Tiny tweaks

c0ffeeca7 reviewed (approved):
Thank you, @gtalarico :+1:
